### PR TITLE
Kotlin floor() method implementation

### LINKED ISSUE #3524: [Term Entry] Kotlin Math .floor()
### Reason/inspiration (optional)

We would like a new term entry on the `.floor()` method for Kotlin: [Math](https://www.codecademy.com/resources/docs/kotlin/math-methods). The entry should go in a new file under `docs/content/kotlin/concepts/math-methods/terms/floor/floor.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by tanbirali**
### Description
A new term entry for Kotlin Floor method.
Close #3524 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/math-methods/terms/floor/floor.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+Title: 'floor()'
+Description: 'The `floor()` method in Kotlin is used to round a floating-point to a nearest lowest integer value'
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/floor/floor.md:3]:
```suggestion
Description: 'Returns the nearest integer that is lower than or equal to the value passed.'
```

File: content/kotlin/concepts/math-methods/terms/floor/floor.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Tags:
+  - 'Arithmetic'
+  - 'Methods'
+  - 'Floor'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`floor()`** function returns a rounded number down to the nearest integer.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/floor/floor.md:15]:
```suggestion
The **`floor()`** function takes a value and returns the nearest integer that is lower than or equal to the value given.
```

File: content/kotlin/concepts/math-methods/terms/floor/floor.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+The result returns a value type of `float`.
+
+## Syntax
+
+```pseudo
+Math.floor(x:Double)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/floor/floor.md:22]:
```suggestion
import kotlin.math.*
floor(x)
```

File: content/kotlin/concepts/math-methods/terms/floor/floor.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+Where `x` is the number to be rounded down to the nearest integer.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/floor/floor.md:25]:
```suggestion
Where `x` is the number of type double to be rounded down to the nearest integer.
```

File: content/kotlin/concepts/math-methods/terms/floor/floor.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+## Example
+
+```kotlin
+println(Math.floor(3.4))
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/floor/floor.md:30]:
```suggestion
import kotlin.math.*
println(floor(3.4))
```

File: content/kotlin/concepts/math-methods/terms/floor/floor.md (hunk 3 above)
Comment (caupolicandiaz) [reply content/kotlin/concepts/math-methods/terms/floor/floor.md:22]: I believe `Math` calls the java library

File: content/kotlin/concepts/math-methods/terms/floor/floor.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+// Output: 3
+```
+
+The result will be a float value of `3` as the function rounds `3.4` down to `3`.
+
+## Codebyte Example
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/floor/floor.md:36]: kotlin is not currently supported by the Codecademy compiler so I would suggest removing this example

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@tanbirali I've added my initial comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by tanbirali** [comment]
@caupolicandiaz I have commited the requested changes and also removed the codebyte example, do let me know if I have to make any more changes.

**Review by caupolicandiaz — APPROVED** [review]
@tanbirali Just added a couple of direct edits to update the example. This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@tanbirali, @caupolicandiaz Looks good.

**Pull request merged**